### PR TITLE
Ashenda Festival 2022: Home and about pages

### PR DESCRIPTION
**Pull request by gtekle**
Added home page
Added about page
Added styling of page section with felxBox layout
Added dynamic content to speakers section in the home page
Added responsive navigation bar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse Inc
+- [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)
```
Comment (DammyShittu) [inline README.md]: - It's really nice that you added the `Acknowledgement` section. 😉 However, kindly acknowledge the author as stated in the requirements.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            <img src="./img/ashenda-icon.png" alt="Ashenda icon" />
+            <div class="logo-text-container">
+              <p class="primary-logo-text">Ashenda Tigrai</p>
+              <p class="secondary-logo-text">Festival 2022</p>
+            </div>
+          </a>
+        </div>
+
+        <!-- desktop menu -->
+        <nav class="nav-menu">
```
Comment (DammyShittu) [inline about.html:50]: Super cool navbar layout. Well done 👍 

I think you missed the little navigation bar at the top. 😀

Let's add it 👇

File: index.html [hunk 3]
```diff
@@ -1,111 +1,230 @@
...
+            <div class="program-icon">
+              <img
+                src="./img/icons/exhibition_white.png"
+                alt="Exhibition icon"
+              />
+            </div>
+            <h3 class="program-title">Exhibition</h3>
+            <p class="program-description">
+              you'll have a chance to see how Ashenda was celebrated from
+              centuries ago.
```
Comment (DammyShittu) [inline index.html:112]: [OPTIONAL]
For the mobile view of the `Programs` section, what do you think about reducing the font-size of the sections to allow them to fit better within their cards? 😉

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @gtekle , 

YOU HAVE DONE AN AMAZING JOB SO FAR 👏 

To highlight,

- your UI design is amazing ✔️ 
- you have a well-structured code base ✔️ 
- you have a nicely written README file ✔️, and
- your presentation video was awesome ✔️ 
 
However, there are some issues that you still need to work on to get this project approved but you are almost there!


# REQUIRED CHANGES ♻️

- The `h1` tag is a really nice tag to use, but it's also very special. Just one `h1` tag is required per page to signify the main header. Kindly go through your HTML files and replace all other `h1` tags with other heading tags(h2 - h6). 😀 

- It would be nice to have some text in the `alt` attribute of the `img` tag. This would improve user experience, especially for people using screen readers, and also if the image fails to load due to one reason or the other.

_Check the comments under the review for more comments and suggestions_

# OPTIONAL SUGGESTIONS 🔢 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- For the mobile view, I noticed that the background color that was given to the navbar when it's in the fixed position almost blends with the background color of most sections of the page. What do you think about adding a darker color as the background color of that navbar? 😉 

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gtekle** [comment]
Hi @DammyShittu
Thank you very much for the detailed feedback. I have fixed both the required and the optional feedback that you gave me because I found all of them valuable.

File: about.html (hunk 2 above)
Comment (moise10r) [reply about.html:50]: This issue is well addressed. Well done! :fireworks: :clap:

File: index.html [hunk 4]
```diff
@@ -1,111 +1,237 @@
...
+          <a href="./index.html" class="logo-link">
+            <img src="./img/ashenda-icon.png" alt="Ashenda icon" />
+            <div class="logo-text-container">
+              <p class="primary-logo-text">Ashenda Tigrai</p>
+              <p class="secondary-logo-text">Festival 2022</p>
+            </div>
+          </a>
+        </div>
+
+        <!-- desktop menu -->
```
Comment (moise10r) [inline index.html:50]: - [OPTIONAL ] Please remove this commented code, It is always a good practice to keep your code clean from such comments.

File: index.html [hunk 5]
```diff
@@ -1,111 +1,237 @@
...
-        </a>
-        <ul class="mobile-nav-list">
-          <li class="nav-link"><a href="#about">Portfolio</a></li>
-          <li class="nav-link"><a href="#program">About</a></li>
-          <li class="nav-link"><a href="#join">Contact</a></li>
-          <li class="nav-link"><a href="#sponsor">About</a></li>
-          <li class="nav-link"><a href="#news">Contact</a></li>
-        </ul>
-      </nav>
+        <!-- mobile nav -->
```
Comment (moise10r) [inline index.html:66]: - [OPTIONAL ] Please remove this commented code, It is always a good practice to keep your code clean from such comments.

File: index.html [hunk 6]
```diff
@@ -1,111 +1,237 @@
...
-      <div class="speaker-info">
-        <h4 class="name">Yochai Benkler</h4>
-        <p class="job-title">Professor of Legal Studies at Harvard Law School</p>
-        <p class="program-description">Benkler studies commons-based peer production, and published his seminal book The Wealth of Networks in 2006. </p>
+    <section class="featured-speakers">
+      <div class="container">
+        <h2 class="section-title">Featured Speakers</h2>
+        <hr class="section-title-underline" />
+        <div class="speaker-container">
+          <!-- content is dynamically generated -->
```
Comment (moise10r) [inline index.html:179]: - [OPTIONAL ] Please remove this commented code, It is always a good practice to keep your code clean from such comments.

**Review by moise10r — APPROVED** [review]
## Hi @Ashenda, 

## **Congratulations! 🎉 Project Approved ✅**

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 👏👏👏 

 #### Optional suggestions
   * N/A

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gtekle** [comment]
Hi @moise10r
Thank you for the feedback. I have removed the comments in the `html` files for the code is self explanatory as per your suggestion.

**Pull request merged**